packages/ui/src/components/alert/AlertRoot.tsx

```diff
@@ -0,0 +1,140 @@
+import { forwardRef, PropsWithChildren, useLayoutEffect, useRef, useState } from 'react'
+
+import { Button } from '@components/button'
+import { Icon, IconNameMap } from '@components/icon'
+import { useResizeObserver } from '@hooks/use-resize-observer'
+import { cn } from '@utils/cn'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+const alertVariants = cva('cn-alert', {
+  variants: {
+    theme: {
+      info: 'cn-alert-info',
+      danger: 'cn-alert-danger',
+      warning: 'cn-alert-warning'
+    }
+  },
+  defaultVariants: {
+    theme: 'info'
+  }
+})
+
+const iconMap: Record<NonNullable<VariantProps<typeof alertVariants>['theme']>, keyof typeof IconNameMap> = {
+  info: 'info-circle',
+  danger: 'cross-circle',
+  warning: 'warning-triangle-outline'
+}
+
+const MAX_HEIGHT = 60
```

Comment:
Playing around with the component, when there's a title in use, only 1 line of the description is shown. I wonder if it makes sense to have the max height at perhaps 70/80 when a title is used and 60 when not?

Reply:
Fixed, check it please